apps/mongodb/workload/mongo_loadgen.yml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: mongo-loadgen
+spec:
+  template:
+    metadata:
+      name: mongo-loadgen
+      labels:
+        loadgen: mongo-loadgen
```

Comment:
@shashank855 can we take these label values from env

Reply:
sure i will make the changes